examples/discovery-search-app/README.md

```diff
@@ -1,5 +1,156 @@
+<!-- START doctoc generated TOC please keep comment here to allow auto update -->
+<!-- DON'T EDIT THIS SECTION, INSTEAD RE-RUN doctoc TO UPDATE -->
+
+**Table of Contents** _generated with [DocToc](https://github.com/thlorenz/doctoc)_
```

Comment:
We should remove these comments and the "generated with ..." section, unless we intend to consistently run the generator.

Reply:
The images in this file don't load. Click on "View file" to see what it looks like rendered.

Reply:
I didn't realize until I committed my changes that doctoc actually runs during the commit hooks. So, in that sense I think that we should keep a mention of DocToc in here, but add to the comments that you don't have to manually run it in order to update the ToC